src/index.spec.ts

```diff
@@ -4,9 +4,114 @@ import {
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
 
-test('Testing time!', () => {
-  console.log('welcome');
+import fetch from 'node-fetch';
+
+describe('Test isInteger() function', () => {
+  test('Should return true', () => {
+    expect(isInteger(123)).toBe(true);
+  });
+
+  test('Should return false', () => {
+    expect(isInteger('str')).toBe(false);
+  });
+});
+
+describe('Test toLowerCase() function', () => {
+  test('Should return a string in lowercase', () => {
+    expect(toLowerCase('sTrInGyWoRd')).toEqual('stringyword');
+  });
+
+  test('Should return an error message', () => {
+    expect(toLowerCase('')).toBe('Please provide a string');
+  });
+});
+
+describe('Test removeDuplicatesFromArray() function', () => {
+  test('Should return an error', () => {
+    expect(() => removeDuplicatesFromArray(null as any)).toThrowError();
+  });
```

Comment:
You can expect the error in `.toThrowError` to be specific, and use `'please provide an array of numbers or strings'` inside this test